### DISCUSSION TITLE
Is it possible to use raw html?

### DISCUSSION THREAD

**Discussion by kittiu** [frappe/print_designer#149] [Q&A] [answered]
1. Print Designer is awsome to be used as designer, but sometime we will require raw html, is this possible?
2. For the body, it seem to be possible, and this really save my life, as some of the table features is not ready yet.

![image](https://github.com/frappe/print_designer/assets/1973598/53d0c7fe-a54b-458f-af9d-12a0ba8b2752)

**Comment by maharshivpatel** [comment, marked as answer]
@kittiu you can set raw html as variable in jinja custom data. then use the variable whenever you want to print it.

https://github.com/frappe/print_designer/pull/115

**Reply by maharshivpatel** [reply]
also, can you share what features you feel seems to be missing for table.

**Comment by kittiu** [comment]
Thank you, got it. The `set send_to_jinja` is what I miss previously.
